source/guide_phpbb.rst

```diff
@@ -0,0 +1,280 @@
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+  * :manual:`mail <mail-access>`
+
+License
+=======
+
+phpBB_ is released under the `GPLv2 License`_.
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.4:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.4'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+If you want to use your phpBB board with your own domain, you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Create Database
+===============
+
+phpBB saves your data in a :manual:`MySQL <database-mysql>` database. It is recommended to use an :manual_anchor:`additional database <database-mysql.html#additional-databases>` (e.g. ``isabell_phpbb``) instead of the default database.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mysql --verbose --execute="CREATE DATABASE ${USER}_phpbb"
+ --------------
+ CREATE DATABASE isabell_phpbb
+ --------------
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
+------
+
+Go to the phpBB download_ website and copy the URL of the latest ZIP release, e.g. ``https://download.phpbb.com/pub/release/3.3/3.3.3/phpBB-3.3.3.zip``.
+
+Then, ``cd`` to your :manual:`document root <web-documentroot>` and download the latest release of phpBB and extract it:
+
+::
+
+ [isabell@stardust ~]$ cd ~/html/
+ [isabell@stardust html]$ curl -L https://download.phpbb.com/pub/release/3.3/3.3.3/phpBB-3.3.3.zip -o phpbb.zip
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  100 25565    0 25565    0     0  22942      0 --:--:--  0:00:01 --:--:-- 22948
+  100 5138k  100 5138k    0     0   769k      0  0:00:06  0:00:06 --:--:-- 1108k
+ [isabell@stardust html]$ unzip phpbb.zip
+ [isabell@stardust html]$ mv phpBB3/* .
+ [isabell@stardust html]$ rm phpbb.zip && rm phpBB3/ -rf
+ [isabell@stardust html]$
+
+Step 2 - GUI
+------------
+
+Now point your browser to your uberspace URL ``https://isabell.uber.space`` and follow the instructions of the Installer Assistant.
+Click on the Tab "Install" to start the installation.
+
+You will need to enter the following information:
+
+  * Administrator username and password: choose a username (maybe not *admin*) and a strong password for the admin user
+  * your MySQL hostname, username, password and database name: the hostname is ``localhost`` and you should know your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>` by now. If you don't, start reading again at the top. You can leave the Database server port empty. As database name insert the name of the created additional database.
```

Comment:
Please don't belittle the user :) Maybe they skipped the "MySQL" stuff because they didn't know what that is, but saw "database" now, which they know.
```suggestion
  * your MySQL hostname, username, password and database name: the hostname is ``localhost``. Check the beginning of the guide for your :manual_anchor:`database credentials <database-mysql.html#login-credentials>`. You can leave the Database server port empty. As database name insert the name of the created additional database, e.g. ``isabell_phpbb``.
```